pennylane/workflow/qnode.py

```diff
@@ -486,14 +486,20 @@ def __init__(
         for kwarg in gradient_kwargs:
             if kwarg in ["gradient_fn", "grad_method"]:
                 warnings.warn(
-                    f"It appears you may be trying to set the method of differentiation via the kwarg "
-                    f"{kwarg}. This is not supported in qnode and will default to backpropogation. Use "
-                    f"diff_method instead."
+                    "It appears you may be trying to set the method of differentiation via the "
+                    f"kwarg {kwarg}. This is not supported in qnode and will default to "

```

Comment:
I would personally use `keyword argument` in a warning, but this is just a matter of taste
```suggestion
                    f"keyword argument {kwarg}. This is not supported in qnode and will default to "

```